logging/log_api.go

```diff
@@ -42,11 +42,11 @@ func Info(args ...interface{}) { DefaultLogger.Info(args...) }
 // Infof is for logging with default logger.
 func Infof(format string, args ...interface{}) { DefaultLogger.Infof(format, args...) }
 
-// Warn is for logging with default logger.
```

Comment:
Could this exported function keep the short version for warning level? Or is there some reasoning behind it?